docs/README.md

```diff
@@ -1,3 +1,17 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Semantic Conventions
+# no_list: true
+cascade:
+  body_class: otel-docs-spec
+  github_repo: &repo https://github.com/open-telemetry/semantic-conventions
+  github_subdir: docs
+  path_base_for_github_subdir: content/en/docs/specs/semconv/
```

Comment:
To some extent, these `path_base` selections are actually implementation details of the docs workflow.

Should this specific value be something inserted by the adjust pages perl script?


I'm worried about keeping path_base up-to-date as we add new markdown files. 

At a minimum, there should be a section under CONTRIBUTING.md about how to fill it out and why it's needed.

At a maximum, I think perhaps the `path_base` or `from` sections should be automatically derived on the docs side.  I'm happy to contribute that.

Reply:
Discussed below.  Since this is a non-breaking, we can accept PR as-is.

We can flesh out CONTRIBUTING.md later.